f/briefing/f_orbatNotes.sqf

```diff
@@ -160,7 +160,10 @@ player createDiaryRecord ["fa3_actions", ["Group Leadership",
 	OVERRIDE GROUP CONTROL: <br/> 
 	 | <execute expression=""
 		if (leader group player != player) then {
-			hintsilent 'Taking control of your group';group player selectLeader player;
+			hintsilent 'Taking control of your group';
+			leader (group player) setUnitRank 'PRIVATE'; 
+			player setUnitRank 'COLONEL';
```

Comment:
Does this change the order of the units e.g. in DUI? Or is that set by the editor / join order?

Reply:
DUI uses join/creation order for its display order.